### PR TITLE
Add CSS compat data for many non-standard CSS properties/selectors

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
Tackles some more of the entries on [#2684](https://github.com/mdn/browser-compat-data/issues/2684#issuecomment-421428277). Properties and selectors involved include:
- -webkit-border-before
- -webkit-box-reflect
- -webkit-mask-composite
- :-webkit-autofill
- ::-webkit-file-upload-button
- ::-webkit-inner-spin-button
- ::-webkit-outer-spin-button
- ::-moz-color-swatch

_Please note: all of the data origins from the existing MDN pages that utilize them, with no validation of the original data's accuracy._

### CODE REVIEW

File: css/selectors/-webkit-autofill.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+  "css": {
+    "properties": {
+      "-webkit-autofill": {
+        "__compat": {
+          "description": "<code>:-webkit-autofill</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:-webkit-autofill",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline css/selectors/-webkit-autofill.json]: `true`

File: css/selectors/-webkit-autofill.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline css/selectors/-webkit-autofill.json]: `true`

File: css/selectors/-webkit-inner-spin-button.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
+{
+  "css": {
+    "properties": {
+      "-webkit-inner-spin-button": {
+        "__compat": {
+          "description": "<code>::-webkit-inner-spin-button</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::-webkit-inner-spin-button",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline css/selectors/-webkit-inner-spin-button.json]: `true`

File: css/selectors/-webkit-inner-spin-button.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline css/selectors/-webkit-inner-spin-button.json]: `true`

File: css/selectors/-webkit-outer-spin-button.json [hunk 5]
```diff
@@ -0,0 +1,58 @@
+{
+  "css": {
+    "properties": {
+      "-webkit-outer-spin-button": {
+        "__compat": {
+          "description": "<code>::-webkit-outer-spin-button</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::-webkit-outer-spin-button",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline css/selectors/-webkit-outer-spin-button.json]: `true`

File: css/selectors/-webkit-outer-spin-button.json [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline css/selectors/-webkit-outer-spin-button.json]: `true`

**Comment by vinyldarkscratch** [comment]
Thanks for review, @Elchi3!  I definitely plan on replacing my editor soon due to the whole newline issue, haha -- everything should now be fixed.  :wink:

**Comment by connorshea** [comment]
@vinyldarkscratch I'd recommend VS Code :P

**Comment by vinyldarkscratch** [comment]
> @vinyldarkscratch I'd recommend VS Code if you're open to suggestions :P

I've tried VS Code but it runs super slow on my computer (and I have a 2017 MacBook Pro) -- I've been using something called Power JSON Editor (mostly for the graphical interface) and Sublime Text for raw code, though if they've improved on optimization I'll probably give VS Code another try!

**Comment by connorshea** [comment]
Odd, I have a late 2013 MacBook Pro and it runs pretty well 🤔 Not as well as Sublime did, but ultimately the superior feature set was more important for me. Whatever works for you! :D

**Pull request merged**